### PR TITLE
fix the examples to point at the oss repository

### PR DESCRIPTION
**Pull request by oliveromahony**
### Proposed changes

Updated the grafana example to use the OSS docker image

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [x] I have read the [`CONTRIBUTING`](https://github.com/nginx/agent/blob/main/docs/CONTRIBUTING.md) document
- [x] If applicable, I have added tests that prove my fix is effective or that my feature works
- [x] If applicable, I have checked that any relevant tests pass after adding my changes
- [x] I have updated any relevant documentation ([`README.md`](https://github.com/nginx/agent/blob/main/README.md))

### CODE REVIEW

**Review by aphralG — APPROVED** [review]

File: examples/grafana-metrics/Makefile [hunk 1]
```diff
@@ -1,3 +1,6 @@
+OS_RELEASE:=ubuntu
```
Comment (dhurley) [inline examples/grafana-metrics/Makefile:1]: These 3 variables aren't being used so that can be removed

Comment (oliveromahony) [reply examples/grafana-metrics/Makefile:1]: I updated the PR to use the variables

**Review by Dean-Coakley — APPROVED** [review]

**Review by dhurley — APPROVED** [review]

File: Makefile [hunk 2]
```diff
@@ -244,5 +249,14 @@ run-container: ## Run container from specified IMAGE_TAG
 		$(CONTAINER_CLITOOL) run ${IMAGE_TAG}
 
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
-# Dashboard Targets                                                                                               #
+# Example Dashboard Targets                                                                                       #
 # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # # #
+clean-example: clean ## Clean example packages and docker
```
Comment (dhurley) [inline Makefile:254]: Can you rename the targets to something more specific like `clean-grafana-metrics-example`?

File: Makefile [hunk 3]
```diff
@@ -244,5 +249,14 @@ run-container: ## Run container from specified IMAGE_TAG
...
+	mkdir -p ../../build
```
Comment (dhurley) [inline Makefile:255]: should this be `mkdir -p ./build`?

**Review by dhurley — APPROVED** [review]

File: Makefile [hunk 4]
```diff
@@ -181,7 +190,7 @@ performance-test: ## Run performance tests
 
 integration-test: local-deb-package
 	PACKAGE_NAME=${PACKAGE_NAME} BASE_IMAGE=${BASE_IMAGE} go test ./test/integration/install
-	PACKAGE_NAME=${PACKAGE_NAME} BASE_IMAGE=${BASE_IMAGE} go test ./test/integration/api
+	# PACKAGE_NAME=${PACKAGE_NAME} BASE_IMAGE=${BASE_IMAGE} go test ./test/integration/api
```
Comment (dhurley) [inline Makefile:193]: need to uncomment this again

**Review by aphralG — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 5]
```diff
@@ -196,6 +196,29 @@ jobs:
           retention-days: 3
 
 
+  build-example:
+    name: Build Grafana Dashboard Example
+    runs-on: ubuntu-22.04
+    concurrency: ubuntu-22.04-${{ matrix.runner.arch }}
```
[the remark this answers is not in the document]
Comment (oliveromahony) [reply .github/workflows/ci.yml]: I removed it for now

**Pull request merged**